tests/subworkflows/ebi-metagenomics/rrna_extraction/nextflow.config

```diff
@@ -0,0 +1,14 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    
+    withName: INFERNAL_CMSEARCH {
+    ext.args = '--cut_ga'
+    }
+}
+
+params {
+
+    rfam = "/hps/nobackup/rdf/metagenomics/service-team/users/chrisata/scripts_taxdb_nf_test/RFAM/14.10/ribo.cm"
```

Comment:
Does it work on github actions? Probably you can add heavy files on our FTP and pull from there?

Reply:
oh good point I forgot about this in the test.. I'll replace it with a small chunk of our Rfam file for now

Reply:
but long term I agree would be good to have these on the FTP

Reply:
fixed this by adding small chunk of rfam as test files